### DISCUSSION TITLE
List of papers involving UniMath

### DISCUSSION THREAD

**Discussion by benediktahrens** [UniMath/UniMath#1705] [Announcements]
The list located at https://github.com/UniMath/UniMath/wiki/Articles-with-accompanying-formalization-in-UniMath is potentially very useful. However, right now I think there are some issues with it:
1. It needs updating: e.g., the construction of the circle in UniMath has been published (https://www.sciencedirect.com/science/article/pii/S0022404921000232)
2. It needs sorting; I would suggest sorting it by date, possibly, from newest to oldest.
3. It needs filtering: e.g., the paper by Menno de Boer is not about UniMath (still a great piece of work, of course!).
4. It needs completing: some entries are missing links to the papers.
5. It needs advertising, e.g., on the README (next to the citation information).

Less urgent, but possibly useful:
- Insert links to DBLP entries for each paper that is listed
- Transform the page into a Markdown file that is compiled into beautiful HTML, similar to https://unimath.github.io/UniMath/

**Comment by m-lindgren** [comment]
Also the thesis by de Boer is a licentiate thesis, not a master thesis, if I'm not mistaken. Or maybe a Swedish licentiate thesis corresponds to a master thesis internationally?

**Reply by mortberg** [reply]
Licentiate is not the same as a master internationally. It's a separate Swedish degree that's pretty much halfway to the PhD (so substantially more than a master)

**Comment by nmvdw** [comment]
It might be convenient to create a bibtex file in the UniMath repository containing all papers, and then this page will be kept consistent with it.

**Comment by nmvdw** [comment]
This is the list of papers/theses that I could find that make use of UniMath in some way [unimath-papers.txt](https://github.com/UniMath/UniMath/files/11644159/unimath-papers.txt).
